include/mqtt/endpoint.hpp

```diff
@@ -1602,14 +1605,22 @@ class endpoint : public std::enable_shared_from_this<endpoint<Socket, Mutex, Loc
      * packet_id is automatically generated.
      */
     packet_id_t publish_at_least_once(
-        std::string const& topic_name,
-        std::string const& contents,
+        std::string topic_name,
+        std::string contents,
         bool retain = false,
         std::vector<v5::property_variant> props = {}
     ) {
-        packet_id_t packet_id = acquire_unique_packet_id();
-        acquired_publish_at_least_once(packet_id, topic_name, contents, retain, std::move(props));
-        return packet_id;
+        auto pTopic    = std::make_shared<std::string>(std::move(topic_name));
+        auto pContents = std::make_shared<std::string>(std::move(contents));
```

Comment:
Could you user snake_case notation for variables?
When I merged your previous PR, I replaced them to `sp_topic` and `sp_contents`.

Reply:
Updated